0944_Delete_Columns_to_Make_Sorted/try_1.cpp

```diff
@@ -0,0 +1,20 @@
+class Solution {
+public:
+    int minDeletionSize(vector<string>& strs) {
+        int n = strs.size();
+        int str_len = strs[0].length();
+        int ans = 0;
+        
+        for (int i=0; i < str_len; i++) {
+            int cur = 0;
+            for (string str : strs) {

```

Comment:
只讀取的話可以直接到記憶體位置拿str，避免一直複製
```suggestion
            for (string &str : strs) {
```